GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace GDAPI.Utilities.Enumerations.GeometryDash
 {
+    // TODO: Rename to ObjectProperty to consistenize terminology
```

Comment:
You should clear this TODO

Reply:
This is for another PR where terminology will be broadly consistenized.

Reply:
ah, ok